InvoiceMaker/Repository/ClientRepository.cs

```diff
@@ -0,0 +1,113 @@
+﻿using InvoiceMaker.Models;
+using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Web;
+
+namespace InvoiceMaker.Repository
+{
+    public class ClientRepository
+    {
+        string connectionString;
+        public ClientRepository()
+        {
+            connectionString = ConfigurationManager.ConnectionStrings["Database"].ConnectionString;
+        }
+
+        public Client GetById (int id)
+        {
+            Client client = new Client(0, null, false);
+            using (SqlConnection connection = new SqlConnection(connectionString))
+            {
+                connection.Open();
+
+                string sql = @"
+          SELECT Id, ClientName, IsActivated
+          FROM Client
+          WHERE Id = @Id  
+          ORDER BY ClientName
+        ";
+                SqlCommand command = new SqlCommand(sql, connection);
+                command.Parameters.AddWithValue("@Id", id);
+                SqlDataReader reader = command.ExecuteReader();
+
+                if (reader.Read())
+                {
+                    int Id = reader.GetInt32(0);
+                    string name = reader.GetString(1);
+                    bool isActivated = reader.GetBoolean(2);
+                    client = new Client(Id, name, isActivated);
+                    return client;
```

Comment:
Instead of returning `client` here, just let execution pass down to the last line of this method.